config/global.ini.php

```diff
@@ -877,6 +877,10 @@
 ; so it can be disabled here if necessary.
 disable_tracking_matomo_app_links = 0
 
+; Force the order of the first table when building segment queries in MySQL. This can be used to override sub-optimal
```

Comment:
I wouldn't use the word 'force' here as it's a hint, I'd suspect the engine doesn't need to respect it and it would also only work when supported, so perhaps something like..

```suggestion
; Add query hint for the order of joined tables when building segment queries in MySQL. This can be used to override sub-optimal
```

Reply:
Good point, I've now added another config option but will avoid the use of force for both :+1: